app/controllers/articles_controller.rb

```diff
@@ -81,6 +81,13 @@ def manage
     @org_members = @organization.users.pluck(:name, :id) if @organization
   end
 
+  def subscriptions
```

Comment:
These subscriptions are `user_subscriptions`, right? Is this controller the right place for them?

Reply:
I think this should be in the `DashboardsController`

cc @atsmith813

Reply:
➕ to moving this into `DashboardsController`...maybe a `subscribed_users` method?